### DISCUSSION TITLE
Vercel Blob - Outgoing Bandwidth Limit

### DISCUSSION THREAD

**Discussion by ph1lb4** [vercel/community#4662] [Feature Question] [answered]
### Question

I have used up the Vercel Blob outgoing bandwidth in a very short amount of time during development and was just unsure if 100MB (or also 1GB on Pro) is realistic?
I used it to upload photos, which take about 5MB in storage and during development, those images are loaded into a Next.js Image component. The outgoing bandwidth was probably used up since I guess there is little caching during development. But even the Pro plans 1GB seems extremely low.
Now my question am I missing something, or am I using it wrong? Is the blob not supposed to be used for images? Since the storage size is 250MB, the outgoing bandwidth would not even allow to show all stored data once.

**Comment by Alex-Painter** [comment]
Hello! I've just come across the same issue - I've hit the 1GB limit in a few days, not realising I wasn't hitting the browser and/or Edge cache when developing. Did you come to a conclusion on this?

**Reply by ph1lb4** [reply]
We switched to S3 since this is not usable with those limits unfortunately.

**Reply by Alex-Painter** [reply]
Are the bandwidth costs when dev-ing more reasonable?

**Comment by wave3llc** [comment]
I'm on a hobby plan and have hit the limit of 100MB in just a day while testing my app making it unusable and unrealistic. Is there any way we could request increasing the limits?

**Reply by SwiichyCode** [reply]
I'm in the same boat as you. I don't think the service is really usable at the moment due to these limitations.

**Reply by Alex-Painter** [reply]
I moved everything to S3 like @ph1lb4 mentioned - their free tier is immense.

**Reply by bennyjien** [reply]
I also hit the limit of 100MB in a day when developing my website. The website is not even done yet.

**Reply by 1977744311** [reply]
me,too

**Comment by iusmandev** [comment]
any solutions to lower the limit without purchasing any plans as for a students It cant be affordable

**Comment by hatchli** [comment]
Same issue here, limit hit within the first day while developing. Going to have to switch to s3, even though the developer experience with Vercel Blob is superior.

**Comment by callmefredcom** [comment]
Indeed, I faced the same issue, I've just refactored my code for S3.

**Comment by csjui** [comment]
Same issue here. Website has recently gone live and have hit the limit almost immediately. Totally unrealistic.

**Reply by csjui** [reply]
Well, just 11 hours later and the bandwidth limit has been increased to 5GB! Probably just lucky timing, but thank you Vercel!

**Comment by leerob** [comment, marked as answer]
Hey everyone. Appreciate your feedback here.

We've raised the limit for free teams to 5GB of data transfer – and the included data transfer on Pro to 10GB.
